db/migrate/20220411175218_create_tasks.rb

```diff
@@ -13,5 +13,8 @@ def change
 
       t.timestamps
     end
+
+    add_index :tasks, [:title, :challenge_id], unique: true
```

Comment:
+ model spec with should matchers, please 🙏

Reply:
@sveredyuk added it here - [add validation](https://github.com/DevChallengeIT/devchallenge-platform/pull/53/commits/40a4aa04b4b07710ba2604fa6bd8b88679e44ae3)